tools/dbtool/src/main/java/com/horizen/DbTool.java

```diff
@@ -0,0 +1,83 @@
+package com.horizen;
+import java.io.File;
+import java.util.*;
+
+import com.horizen.settings.SettingsReader;
+import com.horizen.tools.utils.ConsolePrinter;
+import com.horizen.tools.utils.MessagePrinter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class DbTool {
+    public static final Set<String> storageNames = new HashSet<>(Arrays.asList(
+            "secret",
+            "wallet",
+            "walletTransaction",
+            "walletForgingStake",
+            "walletCswDataStorage",
+            "state",
+            "stateForgerBox",
+            "stateUtxoMerkleTree",
+            "history",
+            "consensusData",
+            "appState1", "appState2", "appWallet1", "appWallet2" // simple app tests
+    ));
+
+    public static void main(String args[]) {
+
+        // initialize log properties since this app uses log4j from sdk libraries
+        // - default name for the log file
+        String logFileName = System.getProperty("java.io.tmpdir") + File.separator + "db_tool.log";
+        System.setProperty("logFilename", logFileName);
+        // - default levels: all in the file and just errors on console
+        System.setProperty("logFileLevel", "all");
+        System.setProperty("logConsoleLevel", "error");
+
+        Logger log = LogManager.getLogger(com.horizen.DbTool.class);
+
+        if (args.length == 0) {
+            log.error("Please provide settings file name as first parameter!");
+            return;
+        }
+        if (!new File(args[0]).exists()) {
+            log.error("File on path " + args[0] + " doesn't exist");
+            return;
+        }
+        String settingsFileName = args[0];
```

Comment:
why not to have as a first parameter just a path to the dir folder of the given node?
Full config file seems too much.

Reply:
Ok, done